wgengine/magicsock/endpoint.go

```diff
@@ -99,6 +92,8 @@ type sentPing struct {
 	at      mono.Time
 	timer   *time.Timer // timeout timer
 	purpose discoPingPurpose
+	res     *ipnstate.PingResult
+	cb      func(*ipnstate.PingResult)
```

Comment:
looks like this can be nil? document a bit here

```
    cb func(*ipnstate.PingResult) // or nil if .....
```

Reply:
Thanks! Fixed.